R/tt_dotabulation.R

```diff
@@ -720,6 +719,18 @@ setMethod(".make_split_kids", "Split",
     SIMPLIFY = FALSE))
 
     inner <- .set_kids_sect_sep(inner, spl)
+    
+    # Trying to split nothing with page_by
+    if (is.null(inner) && !is.na(ptitle_prefix(spl))) {
```

Comment:
The issue here is that there are going to be expectations about tables that come out of applying a layout that has page-by splitting. Most importantly, that there will be page titles for any table that is returned an eventually exported. QC is likely to (correctly) look for them, etc. The tables are "wrong" from the user perspective without them. 

This returns a table that isn't going to be the correct shape, and I am still very much not convinced that is better than returning no table at all and throwing an error. In fact I'm pretty strongly convinced of the opposite.

Reply:
I agree with having the error. Also, later the pagination error for it is very mysterious. That would be solved with sanitization though